go/libkb/api.go

```diff
@@ -322,11 +322,15 @@ var lastUpgradeWarning *time.Time
 func (a *InternalAPIEngine) consumeHeaders(resp *http.Response) error {
 	u := resp.Header.Get("X-Keybase-Client-Upgrade-To")
 	p := resp.Header.Get("X-Keybase-Upgrade-URI")
+	m := resp.Header.Get("X-Keybase-Upgrade-Message")
```

Comment:
base64 is safe, but why not URL encoding so it can still be read that way? do we need to be able to send binary stuff in there?